server/parsec/backend.py

```diff
@@ -52,9 +54,12 @@
         TestbedEvent: TypeAlias = Any  # pyright: ignore[reportRedeclaration]
         TestbedTemplateContent: TypeAlias = Any  # pyright: ignore[reportRedeclaration]
 
+logger = get_logger()
+
 
 @asynccontextmanager
 async def backend_factory(config: BackendConfig) -> AsyncGenerator[Backend, None]:
+    logger.info("Backend configuration", **dataclasses.asdict(config))
```

Comment:
I think it would be better to add a `__str__` method to `BackendConfig`, this would prevent risking leaking sensible values every time we add one to the config !